src/components/pages/Books.js

```diff
@@ -1,13 +1,35 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please 🙏🏽 don't disable linters, instead of fixing them manually, it will help you to use better practices of JS.